RELEASING.md

```diff
@@ -1,33 +1,44 @@
+    - [Single Repo Version Numbers](#single-repo-version-numbers)
   - [Incrementing Versions](#incrementing-versions)
-- [Tagging](#tagging)
-- [Release Labels](#release-labels)
-- [Release Process](#release-process)
-  - [OpenSearch Bundle Minor Releases and Patch Releases](#OpenSearch-Bundle-Minor-Releases-and-Patch-Releases)
-    - [Entrance Criteria to Start Release Window](#Entrance-Criteria-to-Start-Release-Window)
-    - [Exit Criteria to Close Release Window](#Exit-Criteria-to-Close-Release-Window)
-  - [OpenSearch Major Releases](#OpenSearch-Major-Releases)
-  - [Changing Release Date](#Changing-Release-Date)
-- [Security Reviews](#security-reviews)
-- [Backporting](#backporting)
+- [Releases](#releases)
+  - [Major Releases](#major-releases)
+  - [Minor Releases](#minor-releases)
+  - [Patch Releases](#patch-releases)
+- [Releasing](#releasing)
+  - [OpenSearch Releases](#opensearch-releases)
+    - [Release Process](#release-process)
+    - [Release Schedule](#release-schedule)
+    - [Release Window](#release-window)
+      - [Entrance Criteria to Start Release Window](#entrance-criteria-to-start-release-window)
+      - [Exit Criteria to Close Release Window](#exit-criteria-to-close-release-window)
+  - [Other Projects](#other-projects)
+  - [Tagging](#tagging)
+  - [Release Labels](#release-labels)
+  - [Changing Release Dates](#changing-release-dates)
+  - [Requesting a Release](#requesting-a-release)
+  - [Security Reviews](#security-reviews)
 
 ## Overview
 
-This document explains the release strategy for artifacts in this organization.
+This document explains the release strategy for artifacts in this organization, including large distributions, such as OpenSearch with OpenSearch Dashboards, and smaller distributions, such as clients.
 
 ## Branching
 
 Projects create a new branch when they need to start working on 2 separate versions of the product, with the `main` branch being the furthermost release. 
 
-### OpenSearch Branching
+### OpenSearch and OpenSearch Dashboards Branching
 
-[OpenSearch](https://github.com/opensearch-project/OpenSearch) typically tracks 3 releases in parallel. For example, given the last major release of 2.0, OpenSearch in this organization maintains the following active branches.
+Most repos in this organization, including [OpenSearch](https://github.com/opensearch-project/OpenSearch) and [OpenSearch-Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards), typically track 3 releases in parallel. For example, given the last major release of 2.0, OpenSearch and all its components in this organization maintains the following active branches.
 
 * **main**: The _next major_ release, currently 3.0. This is the branch where all merges take place, and code moves fast.
 * * **2.x**: The _next minor_ release. Once a change is merged into `main`, decide whether to backport it to `2.x`.
```

Comment:
where is the branch for next patch release? could we add that, i would suggest to use 2.0.x

Reply:
How will that be helpful?

Reply:
I added

```
* * **2.17**: The _last minor_ release and the _next patch_ release. Once a change is merged into `2.x`, it may be backported to, for example, `2.17` for the next 2.17 release candidate, or as a patch for a `2.12.1`.
```

Reply:
@dblock 

* * **2.x**: The _next minor_ release
`2.17` for the next 2.17 release candidate, or as a patch for a `2.12.1`.


what's the different between _next minor_ releae compare to release candidate?

if 2.x if for next minor release, why could we add 2.17.x for next patch release, then we should use 2.17.1 for release candidate 

i know opensearch is using 2.17.1 for tag. but it is not very common compare to other opensource project.

Reply:
On 2.x you usually have the next minor release. Then eventually we cut 2.17 and make last minute fixes there to make the 2.17 release. So they are the same thing.

Then we make patches on top of 2.17 in the 2.17 branch while 2.x will have moved to 2.18. And release 2.17.1 from 2.17 branch.

Suggest a change if that's not clear from my text?